server/lib/rules/authorize.js

```diff
@@ -45,6 +45,14 @@ function (user, context, callback) {
 <% } else { %>
     return callback(null, user, context);
 <% } %>  });
+  
+  // Convert groups to array
+  function parseGroups(data) {
+    if (typeof data === 'string') {
+      return data.replace(/,/g, ' ').replace(/\\s+/g, ' ').split(' ');
```

Comment:
could/should this be simplified as `data.split(/\s+|,/)`?

either way, please add a comment here for future about splitting on comma or any spaces

Reply:
`split(/\s+|,/)` wont work for "Group1, Group2".
```
> "Group1, Group2".split(/\s+|,/);
< (3) ["Group1", "", "Group2"]
```